documentation/source/install/install_usecases.rst

```diff
@@ -354,3 +263,40 @@ It may take some time for the pods to start as the images must be pulled and ins
 The jupyter server dashboard can be accessed on the Kubernetes node port as in the following screen.
 
 .. image:: https://raw.githubusercontent.com/openebs/openebs/master/documentation/source/_static/Jupyter.png
+
+Crunchy Postgres
+=================
+ 
+Running Crunchy Postgres on OpenEBS
+------------------------------------
+
+The following steps bring up a postgresql stateful set with one master and one replica on OpenEBS storage. OpenEBS has used centos-based postgresql containers from crunchy-data to achieve the same.
```

Comment:
Maybe, "This example uses centos-based postgresql containers from crunchy data to illustrate the same."

Reply:
Done!